### DISCUSSION TITLE
remove unwanted edges in table finder

### DISCUSSION THREAD

**Discussion by joedimagio** [jsvine/pdfplumber#998] [Ask for help with specific PDFs]
I am trying to get rid of the 2 extra vertical lines in the second column which overlap the text and retain only the ones which are actually separating the columns.
```
        table_settings = {
            "vertical_strategy": "text",
            "horizontal_strategy": "explicit",
            "explicit_horizontal_lines": edges,
            "intersection_x_tolerance": 30,
            "intersection_y_tolerance": 200,
        }
    
        im1.debug_tablefinder(table_settings)
```
I assume because of all the empty space in that second column, it thinks there is a separation. 
Cant use min_words_vertical as the text in the column is not predictable.
Is there any other way to remove these lines?

![test-file](https://github.com/jsvine/pdfplumber/assets/5310515/808745e8-a58a-4cff-a9f0-9dc94efc6cb1)

**Comment by samkit-jain** [comment]
Hi @joedimagio Appreciate your interest in the library. You have a few ways around this
1. Explicitly define the vertical separators
  a) You can define the separators yourself like `"explicit_vertical_lines": [100, 200, 300]`. This may work for this PDF but not for others.
  b) You can find the separators using your own logic. A good reference point might be the code [here](https://github.com/jsvine/pdfplumber/discussions/975#discussioncomment-6851683).
2. Play around with the other properties. One of importance could be `min_words_vertical` having a high value.
